resharper/src/resharper-unity/Rider/UnityPluginDetector.cs

```diff
@@ -193,69 +129,23 @@ private InstallationInfo ExistingInstallation(List<FileSystemPath> pluginFiles)
                 myLogger.Warn("Plugin files do not have parent directory (?).");
                 return new InstallationInfo(false, FileSystemPath.Empty, pluginFiles, ZeroVersion);
             }
-
-            // v1.8 is two files, v1.9 is one
-            if (pluginFiles.Count == 0 || pluginFiles.Count > 2)
-            {
-                myLogger.Warn("Unsupported plugin file count: {0}", pluginFiles.Count);
-                return new InstallationInfo(false, FileSystemPath.Empty, pluginFiles, ZeroVersion);
-            }
             
             var pluginDir = parentDirs[0];
-            var filenames = pluginFiles.Select(f => f.Name).ToList();
 
-            // v1.9.0+
             if (pluginFiles.Count == 1)
             {
-                if (pluginFiles[0].Name == MergedPluginFile)
+                if (pluginFiles[0].Name == PluginDllFile)
```

Comment:
It looks like we won't be able to upgrade from `Unity3DRider.cs` to `.dll` this way.